src/main/java/com/github/fishio/control/AchievementScreenController.java

```diff
@@ -63,67 +59,55 @@
 	@FXML
 	private Label icon25;
 	
+	public static void onEnemyKill(int level) {
```

Comment:
Basically, don't use anything static in Controllers. If you need a reference to a controller, use ``(AchievementScreenController) Preloader.loadScreen("achievementScreen").getProperties().get("Controller");``

<review